js/ui/grid_core/ui.grid_core.data_controller.js

```diff
@@ -252,6 +252,10 @@ export const dataControllerModule = {
 
                     storeLoadOptions.filter = this.combinedFilter(storeLoadOptions.filter);
 
+                    if(storeLoadOptions.filter?.length === 1 && storeLoadOptions.filter[0] === '!') {
+                        e.data = { data: [], totalCount: 0 };
```

Comment:
why object instead of an empty array?

Reply:
In case `remoteOperations: true` datagrid raise error 'no totalCount' if I pass just empty array